src/main.rs

```diff
@@ -14,12 +14,35 @@ use std::io::Error;
 use std::sync::Arc;
 use std::thread;
 use std::thread::JoinHandle;
+
+use ::tonic::transport::Uri;
+use clap::Parser;
 use storage::content_manager::toc::TableOfContent;
 
 use crate::common::helpers::create_search_runtime;
 use crate::settings::Settings;
 
+/// Qdrant (read: quadrant ) is a vector similarity search engine.
+/// It provides a production-ready service with a convenient API to store, search, and manage points - vectors with an additional payload.
+///
+/// This CLI starts a Qdrant peer/server.
```

Comment:
This part will be printed on `qdrant --help` also version and optional args will be printed